### PR TITLE
PLNSRVCE-1598 update module and reference after moving the repo to konflux-ci org

### PR DESCRIPTION
**Pull request by jkhelil**
- Update references for old repo to ne org repo konflux-ci

### CODE REVIEW

**Comment by avinal** [comment]
I think you missed this: https://github.com/konflux-ci/sprayproxy/blob/c6f8bf26007c140d51326760e16eb52ae5072e3b/.tekton/pull-request.yaml#L17

**Review by avinal — COMMENTED** [review]
I think you missed this: https://github.com/konflux-ci/sprayproxy/blob/c6f8bf26007c140d51326760e16eb52ae5072e3b/.tekton/pull-request.yaml#L17

File: .tekton/push.yaml [hunk 1]
```diff
@@ -14,7 +14,7 @@ spec:
     - name: revision
       value: "{{revision}}"
     - name: output-image
-      value: "quay.io/redhat-appstudio/sprayproxy:{{revision}}"
+      value: "quay.io/konflux-ci/sprayproxy:{{revision}}"
```
Comment (enarha) [inline .tekton/push.yaml:17]: Is there konflux-ci org in Quay? It either does not exist or it's me not having permissions to confirm.

Comment (sayan-biswas) [reply .tekton/push.yaml:17]: Even I don't see the `konflux-ci` in quay.

Comment (jkhelil) [reply .tekton/push.yaml:17]: so is the plan to keep the image under redhat-appstudio ?

**Comment by jkhelil** [comment]
@enarha @Roming22 @avinal 
Do you think we can merge this please ?

**Review by Roming22 — APPROVED** [review]
lgtm but as I'm not the most knowledgeable on the sprayproxy I'd like another person to approve.

**Comment by jkhelil** [comment]
@sayan-biswas @avinal Do you approve this please ?

**Comment by enarha** [comment]
The change is LGTM. I do not have permissions to merge though.

**Review by avinal — APPROVED** [review]
Looks good to me

**Pull request merged**